packages/node-core/src/indexer/storeCache/cacheModel.ts

```diff
@@ -290,40 +301,26 @@ export class CachedModel<
   private async historicalMarkPreviousHeightRecordsBatch(
     tx: Transaction,
     setRecords: SetData<T>,
-    removeRecords: Record<string, RemoveValue>
-  ): Promise<any[]> {
+    removeRecords: Record<string, RemoveValue>,
+    sequelize: Sequelize
+  ): Promise<void> {
     const closeSetRecords = Object.entries(setRecords).map(([id, value]) => {
       return {id, blockHeight: value.getFirst().startHeight};
     });
     const closeRemoveRecords = Object.entries(removeRecords).map(([id, value]) => {
       return {id, blockHeight: value.removedAtBlock};
     });
-
     const mergedRecords = closeSetRecords.concat(closeRemoveRecords);
 
-    return Promise.all(
-      mergedRecords.map(({blockHeight, id}) => {
-        return this.historicalModel.update(
-          {
-            __block_range: this.model.sequelize.fn(
-              'int8range',
-              this.model.sequelize.fn('lower', this.model.sequelize.col('_block_range')),
-              blockHeight
-            ),
-          },
-          {
-            hooks: false,
-            transaction: tx,
-            // https://github.com/sequelize/sequelize/issues/15179
-            where: {
-              id,
-              __block_range: {
-                [Op.contains]: (blockHeight - 1) as any,
-              },
-            } as any,
-          }
-        );
-      })
+    await sequelize.query(
+      `UPDATE "tutorials-sum-reward"."${
```

Comment:
Hard coded schema here. You can use `this.model.getTableName()` and it will include schema. I think that's what it's called